MiniApp/Classes/Extensions/URLRequest+Authorization.swift

```diff
@@ -0,0 +1,7 @@
+extension URLRequest {
+    mutating func setAuthorizationHeader(environment: Environment) {
+        if environment.subscriptionKey.count > 0 {
```

Comment:
Using https://developer.apple.com/documentation/swift/string/2946268-isempty would be better

Reply:
Done